src/quartz_api/cmd/main.py

```diff
@@ -1,52 +1,201 @@
+    """Model for the health endpoint response."""
+
+    status: int
 
-match cfg.SOURCE:
-    case "quartzdb":
-        if cfg.DB_URL == "" or cfg.DB_URL is None:
-            raise OSError(f"DB_URL env var is required using db source: {cfg.SOURCE}")
+def _custom_openapi(server: FastAPI) -> dict[str, Any]:
+    """Customize the OpenAPI schema for ReDoc."""
+    if server.openapi_schema:
+        return server.openapi_schema
 
-        def get_db_client_override() -> internal.DatabaseInterface: # noqa: D103
-            return internal.inputs.quartzdb.Client(cfg.DB_URL)
-    case "dummydb":
+    openapi_schema = get_openapi(
+        title=server.title,
+        version=server.version,
+        description=server.description,
+        contact={
+            "name": "Quartz API by Open Climate Fix",
+            "url": "https://www.quartz.solar",
+            "email": "info@openclimatefix.org",
+        },
+        license_info={
+            "name": "MIT License",
+            "url": "https://github.com/openclimatefix/quartz-api/blob/main/LICENSE",
+        },
+        routes=server.routes,
+    )
 
-        def get_db_client_override() -> internal.DatabaseInterface: # noqa: D103
-            return internal.inputs.dummydb.Client()
-    case _:
-        raise ValueError(f"Unknown SOURCE: {cfg.SOURCE}. Expected 'dummydb'.")
+    openapi_schema["info"]["x-logo"] = {"url": "/static/logo.png"}
+    openapi_schema["tags"] = server.openapi_tags
+    server.openapi_schema = openapi_schema
 
-# Dependency inject the desired database client
-server.dependency_overrides[get_db_client] = get_db_client_override
+    return openapi_schema
 
 
 def run() -> None:
     """Run the API using a uvicorn server."""
+    # Get the application configuration from the environment
+    conf = ConfigFactory.parse_file((pathlib.Path(__file__).parent / "server.conf").as_posix())
+
+    # Create the FastAPI server
+    server = FastAPI(
+        version=version("quartz_api"),
+        title="Quartz API",
+        description=__doc__,
+        openapi_tags=[{
+            "name": "API Information",
+            "description": "Routes providing information about the API.",
+        }],
+        docs_url="/swagger",
+        redoc_url=None,
+    )
+
+    # Add the default routes
+    server.mount("/static", StaticFiles(directory=static_dir.as_posix()), name="static")
+
+    @server.get("/health", tags=["API Information"], status_code=status.HTTP_200_OK)
+    def get_health_route() -> GetHealthResponse:
+        """Health endpoint for the API."""
+        return GetHealthResponse(status=status.HTTP_200_OK)
+
+    @server.get("/favicon.ico", include_in_schema=False)
+    def favicon() -> FileResponse:
+        """Serve the favicon."""
+        return FileResponse(static_dir / "favicon.ico")
+
+    @server.get("/docs", include_in_schema=False)
+    def redoc_html() -> FileResponse:
+        """Render ReDoc HTML."""
+        return FileResponse(static_dir / "redoc.html")
+
```

Comment:
is the /swagger endpoint still there?

Reply:
Yeah swagger is still at the default `swagger`, as is the redoc docs at `/docs`